mkdocs_rss_plugin/util.py

```diff
@@ -620,13 +620,14 @@ def guess_locale(mkdocs_config: Config) -> str or None:
             return mkdocs_config.get("locale")
 
         # Some themes implement a locale or a language setting
-        if "theme" in mkdocs_config and "locale" in mkdocs_config.get("theme"):
-            locale = mkdocs_config.get("theme")._vars.get("locale")
-            return f"{locale.language}-{locale.territory}"
-        elif "theme" in mkdocs_config and "language" in mkdocs_config.get("theme"):
-            return mkdocs_config.get("theme")._vars.get("language")
-        else:
-            return None
+        if "theme" in mkdocs_config:
+            theme = mkdocs_config["theme"]
+            if "locale" in theme:
+                locale = theme["locale"]
+                return f"{locale.language}-{locale.territory}"
+            elif "language" in theme:
+                return theme["language"]
```

Comment:
As Mkdocs is now using custom objects (stille inhretid from UserDict), I suggest to use attributes getter as done in  https://github.com/Guts/mkdocs-rss-plugin/pull/195 and https://github.com/Guts/mkdocs-rss-plugin/pull/211:

```suggestion
            if "locale" in mkdocs_config.theme:
                locale = theme.locale
                return f"{locale.language}-{locale.territory}"
            elif "language" in mkdocs_config.theme:
                return theme.language
```